### PR TITLE
Add ARM64 Builds For Node.js

### PR DESCRIPTION
**Pull request by kolanos**

### CODE REVIEW

File: nodejs/publish-layers.sh [hunk 1]
```diff
@@ -5,77 +5,178 @@ set -Eeuo pipefail
...
 }
 
-function build-nodejs12x {
-    echo "Building new relic layer for nodejs12.x"
-    rm -rf $BUILD_DIR $NJS12X_DIST
+function build-nodejs12x-arm64 {
+echo "Building new relic layer for nodejs12.x (arm64)"
+    rm -rf $BUILD_DIR $NJS12X_DIST_ARM64
+    mkdir -p $DIST_DIR
+    npm install --prefix $BUILD_DIR newrelic@latest @newrelic/aws-sdk@latest
```
Comment (mrickard) [inline nodejs/publish-layers.sh:67]: The Node agent team's just removed the @newrelic/aws-sdk dependency as a separate install. It's included within the agent now.

File: nodejs/publish-layers.sh [hunk 2]
```diff
@@ -95,39 +196,123 @@ function publish-nodejs12x {
...
     rm -rf $BUILD_DIR $EXTENSION_DIST_DIR $EXTENSION_DIST_PREVIEW_FILE
-    echo "Build complete: ${NJS14X_DIST}"
+    echo "Build complete: ${NJS14X_DIST_ARM64}"
+}
+
+function build-nodejs14x-x86 {
+echo "Building new relic layer for nodejs14.x (x86_64)"
+    rm -rf $BUILD_DIR $NJS14X_DIST_X86_64
+    mkdir -p $DIST_DIR
+    npm install --prefix $BUILD_DIR newrelic@latest @newrelic/aws-sdk@latest
```
Comment (mrickard) [inline nodejs/publish-layers.sh:216]: Same as above: we don't need the @newrelic/aws-sdk as an explicit install here.

**Review by MattWhelan — APPROVED** [review]
I'd really like to see this refactored at some point. We have a huge amount of code duplication in these build scripts, and it's getting unwieldy.

**Review by mrickard — CHANGES_REQUESTED** [review]
Let's remove @newrelic/aws-sdk from the installation lines.

**Review by mrickard — APPROVED** [review]

**Pull request merged**